.github/workflows/master.yml

```diff
@@ -508,21 +508,31 @@ jobs:
           curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
           sudo install kubectl /usr/local/bin/kubectl
           kubectl version --client=true
+      - uses: actions/setup-go@b22fbbc2921299758641fab08929b4ac52b32923
+        with:
+          go-version: ${{env.GO_VERSION}}
+          stable: true
       # conntrack is required for kubernetes 1.18 and higher
       # socat is required for kubectl port forward which is used in some tests such as validateHelmTillerAddon
+      # cri-dockerd is required for Kubernetes 1.24 and higher for none driver
       - name: Install tools for none
         shell: bash
         run: |
           sudo apt-get update -qq
           sudo apt-get -qq -y install conntrack
           sudo apt-get -qq -y install socat
+          git clone -n https://github.com/Mirantis/cri-dockerd
+          cd cri-dockerd
+          git checkout a4d1895a2659ea9974bd7528a706592ab8b74181
```

Comment:
this should be updated to the new version (or maybe we should automate it somehow?)

Reply:
Ah, good catch, I'll make a script to update it like the we go for other things.

This one is more important because the file struct changed so some manual intervention is needed here regardless

Reply:
Updated commit numbers